src/pages/BrandPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Product from '../components/product';
+
+const BrandPage = () => {
+  const params = useParams();
+  const brandNames = useSelector((state) => state.brands.brands);
+  const products = brandNames[params.brandName] || [];
+
+  return (
+    <div
+      className="products"
+      style={{ backgroundImage: 'url(/assets/makeup.jpg)' }}
+    >
+      <p style={{ display: 'none' }}>{params.brandName}</p>
+      {products.map((product) => (
+        <Product details={product} id={product.id} key={product.id} />
+      ))}
+    </div>
+  );
+};
+export default BrandPage;
```

Comment:
- [ ] It would be nice if you have a back button on all your detail pages or BrandPage components header that way the user is easily taken to the previous page. Currently, one has to click the back arrow on the browser to go back or click on the home icon to go back to the main home page and that is not a good user experience. 
- [ ] Kindly add link to take the user back to the previous page without clicking the browser's back arrow 👍 

**Example Screen 1**
![image](https://user-images.githubusercontent.com/50941074/175125736-f1239069-7244-432e-a2ef-82a28f3f54a2.png)

**Example Screen 2**
![image](https://user-images.githubusercontent.com/50941074/175125977-6a8d2cb0-fe75-419a-9200-5b64d8d303f3.png)